### PR TITLE
feat: Inclusion proofs and signature validation

### PR DESCRIPTION
**Pull request by metacertain**

### CODE REVIEW

**Review by mrekucci — COMMENTED** [review]
Would be nice to have some tests.

File: src/PostageStamp.sol [hunk 1]
```diff
@@ -93,6 +97,9 @@ contract PostageStamp is AccessControl, Pausable {
 
     // Lottery pot at last update.
     uint256 public pot;
+    
+    // blocks in 24 hours ~ 24 * 60 * 60 / 5 = 17280
```
Comment (significance) [inline src/PostageStamp.sol:101]: perhaps should add blocktime as a init var and calculate accordingly

File: src/PriceOracle.sol [hunk 2]
```diff
@@ -24,7 +24,7 @@ contract PriceOracle is AccessControl {
     uint256 public currentPrice = minimumPrice;
 
     // Constants used to modulate the price, see below usage
-    uint256[] public increaseRate = [0, 1069, 1048, 1032, 1024, 1021, 1015, 1003, 980];
```
Comment (significance) [inline src/PriceOracle.sol:27]: we should address this in a separate PR

**Comment by nugaon** [comment]
Continuing in #152

**Pull request closed without merging**